labtool2.0/src/components/pages/Notification.js

```diff
@@ -25,13 +25,7 @@ export class Notification extends React.Component {
 
     if (message === undefined) {
       return (
-        <Grid>
-          <Grid.Row>
-            <br />
-            <br />
-            <br />
-          </Grid.Row>
-        </Grid>
+        <div />
```

Comment:
Any reason to return `<div />` instead of null?

Reply:
It adds a bit of a gap where the notification would be, which makes the UI look somewhat better.

Reply:
There would probably be a better way to do it, but let's think about it later